--- a/drivers/usb/dwc3/dwc3_msm_kretprobe.c
+++ b/drivers/usb/dwc3/dwc3_msm_kretprobe.c
@@ -382,10 +382,10 @@ static int dwc3_tp_register_all(void)
 	REG_TP(dwc3_alloc_request,     dwc3_tp_log_request_cb)
 	REG_TP(dwc3_gadget_ep_cmd,     dwc3_tp_log_gadget_ep_cmd_cb)
-	REG_TP(dwc3_log_trb,           dwc3_tp_log_trb_cb)
-	REG_TP(dwc3_log_event,         dwc3_tp_log_event_cb)
-	REG_TP(dwc3_log_ep,            dwc3_tp_log_ep_cb)
+	REG_TP(FIXME_TRB,              dwc3_tp_log_trb_cb)
+	REG_TP(dwc3_event,             dwc3_tp_log_event_cb)
+	REG_TP(FIXME_EP,               dwc3_tp_log_ep_cb)
 #undef REG_TP
 	return 0;
 }
@@ -397,10 +397,10 @@ static void dwc3_tp_unregister_all(void)
 	unregister_trace_dwc3_alloc_request(
 			dwc3_tp_log_request_cb, NULL);
 	unregister_trace_dwc3_gadget_ep_cmd(
 			dwc3_tp_log_gadget_ep_cmd_cb, NULL);
-	unregister_trace_dwc3_log_trb(
+	unregister_trace_FIXME_TRB(
 			dwc3_tp_log_trb_cb, NULL);
-	unregister_trace_dwc3_log_event(
+	unregister_trace_dwc3_event(
 			dwc3_tp_log_event_cb, NULL);
-	unregister_trace_dwc3_log_ep(
+	unregister_trace_FIXME_EP(
 			dwc3_tp_log_ep_cb, NULL);
 	tracepoint_synchronize_unregister();
 }
-- 
2.34.1
